main.go

```
@@ -28,6 +29,17 @@ func init() {
 	// +kubebuilder:scaffold:scheme
 }
 
+func getRedisCLI(log *logr.Logger) *rediscli.RedisCLI {
+	cli := rediscli.NewRedisCLI(log)
+	user := os.Getenv("REDIS_USERNAME")
+	if user != "" {
+		cli.Auth = &rediscli.RedisAuth{
+			User: user,
```

where is the REDISCLI_AUTH (password) usage ?

---

Same answer as here:
https://github.com/PayU/redis-operator/pull/106#discussion_r660268474

---

i understand now. so i would like to prevent from the operator to start when REDIS_USERNAME was supplied but  
REDISCLI_AUTH env variable is not set. you can throw an error (with log) here in case this apply